Bio.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  <meta charset="utf-8">
+  <title>Lebron James Portfolio</title>
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Rock+Salt" rel="stylesheet">
+  </head>
+  
+  <body>
+
+    <nav>
+      <strong>Bio</strong>
+      |
+      <a href="Experience.html">Experience</a>
+     </nav>
+
+    <h1>Who am I?</h1> 
+
+    <hr>
+
+    <img class="witnesses" src="images/WeAreAllWitnesses.jpg">
+
+    <h3>Professional Basketball Player</h3>
+
+    <p>
+      Standing 6 feet 8 inches and weighing 250 pounds, I Lebron James, from Akron Ohio am a superstar guard/foward for the Cleveland Cavaliers. Drafted first overall in 2003.  
+    </p>
+
+    <hr>
+
+    <img src="images/ipromise.jpg">
+
+    <nav>
+      <a href="https://facebook.com/LeBron">Facebook</a>
```

Comment:
The common convention for external links is to open them up in a new tab/window. 🔨